backend/prompt_studio/prompt_studio_core_v2/views.py

```diff
@@ -437,13 +437,21 @@ def fetch_contents_ide(self, request: HttpRequest, pk: Any = None) -> Response:
                     file_path += file_name
                 contents = FileManagerHelper.fetch_file_contents(file_system, file_path)
         else:
-            contents = PromptStudioFileHelper.fetch_file_contents(
-                file_name=file_name,
-                org_id=UserSessionUtils.get_organization_id(request),
-                user_id=custom_tool.created_by.user_id,
-                tool_id=str(custom_tool.tool_id),
-            )
-
+            try:
+                contents = PromptStudioFileHelper.fetch_file_contents(
+                    file_name=file_name,
+                    org_id=UserSessionUtils.get_organization_id(request),
+                    user_id=custom_tool.created_by.user_id,
+                    tool_id=str(custom_tool.tool_id),
+                )
+            except FileNotFoundError:
+                return Response(
+                    {
+                        "data": f"File {file_name} not found. Looks like it "
+                        "must be deleted or moved to a different location."
+                    },
+                    status=status.HTTP_404_NOT_FOUND,
+                )
```

Comment:
@harini-venkataraman please raise a DRF exception instead so that the error message's format follows our conventions for easy parsing in the FE

NIT: Also try to single quote the file name for readability
```suggestion
                return Response(
                    {
                        "data": f"File '{file_name}' not found. Looks like it "
                        "must be deleted or moved to a different location."
                    },
                    status=status.HTTP_404_NOT_FOUND,
                )
```

Reply:
Raised the default `FileNotFound` error being used throughout the app